qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -27,7 +27,7 @@
 logger = logging.getLogger(__name__)
 
 
-class PegasosQSVC(SVC, SerializableModelMixin):
+class PegasosQSVC(ClassifierMixin, SerializableModelMixin):
     """
     This class implements Pegasos Quantum Support Vector Classifier algorithm developed in [1]
     and includes overridden methods ``fit`` and ``predict`` from the ``SVC`` super-class. This
```

Comment:
The class docstring ought to be updated since it had/has `and includes overridden methods ``fit`` and ``predict`` from the ``SVC`` super-class`

Reply:
Thank you for noticing! I fixed it.